readthedocs/organizations/tasks.py

```diff
@@ -15,7 +15,7 @@ def mark_organization_assets_not_cleaned(build_pk):
     try:
         build = Build.objects.get(pk=build_pk)
     except Build.DoesNotExist:
-        log.info("Build does not exist.", build_pk=build_pk)
+        log.debug("Build does not exist.", build_pk=build_pk)
```

Comment:
This seems like it should be a `warning` not an `debug`? Is there a reason we'd expect this to fail?

Reply:
Not really. There is no way to trigger a task with a build_pk and that task not finding the Build object. I would remove the try/except clause for this case, actually.

This protection scenario used here, is useful for other db objects, like Project or Version. However, for Build it does not make sense in the way we use it: "they are never deleted"